test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -47,13 +47,21 @@ void InterpreterState::dumpTraceAndState(ostream& _out) const
 	for (auto const& line: trace)
 		_out << "  " << line << endl;
 	_out << "Memory dump:\n";
-	for (size_t i = 0; i < memory.size(); i += 0x20)
+	u256 curOffset;
```

Comment:
```suggestion
map<u256, u256> words;
for (auto const& [offset, value]: memory)
  words[offset / 0x20] |= value << (256 - 8 - 8 * size_t(offset % 0x20));
for (auto const& [offset, value]: words)
  _out << "  " << std::hex << std::setw(4) << offset << ": " << h256(value).hex() << endl;
```
?
That's probably a tad slower, but much easier to read... we could even go with
``words[offset / 0x20] |= value << (8 * size_t(offset % 0x20));`` and care about endianness only when outputting in the second loop, that would make it even easier to read I guess.